sonic-net/sonic-buildimage
# [BFN] Remove not common entries from pcie yaml configuration

#### Why I did it
Default pcieutil uses one configuration for all models of platform
#### How I did it
Take the configuration file as base for all models of concrete platform where model-specific devices are not listed in this configuration
#### How to verify it
Run pmon#pcied and verify that there is no error/warning logs on initialization step
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
